Docs: Add explanation for rotation layer (#1724)
grafana/oncall: docs/sources/calendar-schedules/web-schedule/create-schedule/index.md

multiple users results in all users in the group being included in corresponding shifts.
1. When you’re satisfied with the rotation preview, click **Create**.

> Oncall supports multiple **layer** for rotation which allows for overlapping schedule.

Reviewer: ```suggestion
Oncall supports multiple **layers** for rotations which allow for overlapping shifts.
```

> Layer defines the schedule priority, for example *Layer 2* rotation override *Layer 1* rotation.

Reviewer: ```suggestion
A layer defines the shift priority, for example a *Layer 2* shift overrides a *Layer 1* shift.
```

> In this case, users under *Layer 1* would not receive notification during the overlapping time with users under *Layer 2*.

Reviewer: ```suggestion
In this case, users in a *Layer 1* shift will not be notified during the time overlapping with a shift in *Layer 2*.
```
